### PR TITLE
chore(sysinst-pt): Add support to Portuguese for sysinst

### PR DESCRIPTION
**Pull request by danielsoro**

### CODE REVIEW

File: usr.sbin/sysinst/msg.entropy.pt [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+message Configure_entropy	{Configurar entropy}
+
+message continue_without_entropy	{Agora nao, continue!}
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.entropy.pt:31]: I think you're missing something here? ;)

Comment (TheFakeMontyOnTheRun) [reply usr.sbin/sysinst/msg.entropy.pt:31]: Oh wait - it's that issue with ã, right?

File: usr.sbin/sysinst/msg.mbr.pt [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+message biosguess
+{Usando as informa�oes existente no disco, meu melhor palpite para a geometria
+do BIOS � %d cilindros %d setores %d cabe�alhos\n}
+
+message realgeom
+{real geom: %d cilindros, %d cabe�alhos, %d setores  (OBS: apenas para compara�ao)\n}
+
+message biosgeom
+{BIOS geom: %d cilindros, %d cabe�alhos, %d setores\n}
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.mbr.pt]: remark: I would say that, for this context, the correct translation for "head" would be "cabeças", not "cabeçalhos". But that's just my opinion man

File: usr.sbin/sysinst/msg.entropy.pt (hunk 1 above)
Comment (danielsoro) [reply usr.sbin/sysinst/msg.entropy.pt:31]: Yeah, exactly that. `ã` and `õ` are not working as expected, so I removed the `~` from the words.

File: usr.sbin/sysinst/msg.mbr.pt (hunk 2 above)
Comment (danielsoro) [reply usr.sbin/sysinst/msg.mbr.pt]: Yeah, I thought the term 'cabeçalho' could be better, but looking at some online documentation about Hard Drivers, the expression is really: 'cabeça'. I'll change it.

File: libexec/ld.elf_so/arch/hppa/hppa_reloc.c [hunk 3]
```diff
@@ -625,6 +625,10 @@ _rtld_relocate_plt_lazy(Obj_Entry *obj)
 		Elf_Addr *where = (Elf_Addr *)(obj->relocbase + rela->r_offset);
 		Elf_Addr func_pc, func_sl;
 
+		/* skip R_PARISC_NONE entries */
+		if (ELF_R_TYPE(rela->r_info) == R_TYPE(NONE))
+			continue;
+
```
Comment (TheFakeMontyOnTheRun) [inline libexec/ld.elf_so/arch/hppa/hppa_reloc.c]: question: I see you're adding more stuff. Is this a merge from newer revisions or are you, in fact, fixing stuff?

File: usr.sbin/sysinst/msg.mbr.pt [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+{Voc� nao marcou uma parti�ao como ativa. Isso pode fazer com que o seu 
+sistema nao inicie corretamente.}
+
+message fixactivepart
+{
+A parti�ao NetBSD do disco deve ser marcada como ativa?}
+
+message setbiosgeom
+{
+Voc� ser� solicitadoa informar a geometria.
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.mbr.pt:55]: remark: you might have missed a space here

Comment (TheFakeMontyOnTheRun) [reply usr.sbin/sysinst/msg.mbr.pt:55]: And two lines below as well ("queo BIOS")

File: usr.sbin/sysinst/msg.mbr.pt [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+}
+
+message wmbrfail
+{A reescrita do MBR falhou. Nao consigo continuar.}
+
+message mbr_inside_ext
+{A parti�ao precisa come�ar dentro da parti�ao estendida}
+
+message mbr_ext_nofit
+{A parti�ao estendida deve ser capaz de conter todas as parti��es contidas}
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.mbr.pt:112]: remark: you end up using a õ here.

Comment (TheFakeMontyOnTheRun) [reply usr.sbin/sysinst/msg.mbr.pt:112]: (or the problem is only with ã?)

File: usr.sbin/sysinst/msg.pm.pt [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+message edit_parts {Editar parti�oes}
+message switch_parts {Switch partitioning scheme}
+message fmtasraid {Formatar como RAID}
+message fmtaslvm {Formatar como LVM PV}
+message encrypt {Criptografar (CGD)}
+message erase {Apaga com seguran�a}
+message undo {Desfazer mudan�as}
+message unconfig {Desconfigurar}
+message edit {Editar}
+message doumount {Desmontar for�ado}
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.pm.pt:166]: suggestion: wouldn't "à força" work better here?

File: usr.sbin/sysinst/msg.pm.pt [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+message lvmlv_readahsect_ask {ReadAheadSectors?}
+message lvmlv_stripes_ask {Stripes?}
+
+message notsupported {Opera�ao nao suportada!}
+message edit_parts {Editar parti�oes}
+message switch_parts {Switch partitioning scheme}
+message fmtasraid {Formatar como RAID}
+message fmtaslvm {Formatar como LVM PV}
+message encrypt {Criptografar (CGD)}
+message erase {Apaga com seguran�a}
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.pm.pt:162]: question: aren't you inferring too much meaning from this?

File: usr.sbin/sysinst/msg.pm.pt [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+message create_vg {Criar grupo de volume (LVM VG)}
+message create_lv {Criar volume l�gico}
+message create_raid {Criar RAID por software}
+message updpmlist {Atualizar lista de dispositivos}
+message savepm {Salvar mudan�as}
+message pmblocked {BLOQUEADO}
+message pmunchanged {SEM USO}
+message pmused {USADO}
+message pmmounted {(montado)}
+message pmunused {(sem usar)}
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.pm.pt:58]: suggestion: perhaps "sem uso" would sound a bit better?

File: usr.sbin/sysinst/msg.pm.pt [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+message finishpm {Finalizar o particionamento}
+message limitcount {Limite de contagem de dispositivos foi atingido!}
+message invaliddev {Dispositivo inv�lido}
+message avdisks {Discos dispon�veis:}
+message nofreedev {N�o � poss�vel alocar o n� do dispositivo!}
+message partman_header
+{Este � o gerenciador de parti��es estendido. Todos os discos, parti��es, etc. estao
+listados abaixo. Para usar RAID, LVM ou CGD:
+1) Crie parti��es com o tipo apropriado;
+2) Crie RAID/LVM VG/CGD usando essas parti��es; 3) Salve as altera��es;
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.pm.pt:70]: question: is the line break important here for option 3?

File: usr.sbin/sysinst/msg.pm.pt [hunk 10]
```diff
@@ -0,0 +1,167 @@
...
+message raid_superpar_ask {SUsPerParityUnit?}
+message raid_superrec_ask {SUsPerReconUnit?}
+message raid_disks {Discos em RAID:}
+message vnd_err_menufmt {CAMINHO NAO DEFINIDO!}
+message vnd_assign {ATRIBUIR}
+message vnd_path_fmt {Caminho de arquivo}
+message vnd_assign_fmt {Criar nova imagem}
+message vnd_size_fmt {Tamanho}
+message vnd_ro_fmt {Somente leitura}
+message vnd_geom_fmt {Defini geometria manualmente}
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.pm.pt:104]: suggestion: a passive voice might make more sense here?

File: libexec/ld.elf_so/arch/hppa/hppa_reloc.c (hunk 3 above)
Comment (danielsoro) [reply libexec/ld.elf_so/arch/hppa/hppa_reloc.c]: I didn't add this stuff, not sure why this PR is getting these other commits when it says that the branch is just 3 commits ahead. I'll try to fix it and if not possible, I open a new PR.

File: usr.sbin/sysinst/msg.pm.pt (hunk 9 above)
Comment (danielsoro) [reply usr.sbin/sysinst/msg.pm.pt:70]: I can add this, but in the English version it doesn't have this structure: https://github.com/danielsoro/netbsd-src/blob/ddafc8c6412feff5b2f20bbd26ddd0a7c4e3853c/usr.sbin/sysinst/msg.pm.en#L68-L71

Guess it should follow the same structure as in the English version.

File: usr.sbin/sysinst/msg.pm.pt (hunk 7 above)
Comment (danielsoro) [reply usr.sbin/sysinst/msg.pm.pt:162]: Inferring!? What do you mean?

File: usr.sbin/sysinst/msg.mbr.pt (hunk 5 above)
Comment (danielsoro) [reply usr.sbin/sysinst/msg.mbr.pt:112]: Yes, same problem. I'll fix it.

File: usr.sbin/sysinst/msg.pm.pt (hunk 7 above)
Comment (TheFakeMontyOnTheRun) [reply usr.sbin/sysinst/msg.pm.pt:162]: Going from simply "Erase" to "Safely erase"?

Comment (danielsoro) [reply usr.sbin/sysinst/msg.pm.pt:162]: The English version is: Safe erase. (https://github.com/danielsoro/netbsd-src/blob/trunk/usr.sbin/sysinst/msg.pm.en#L162)

File: usr.sbin/sysinst/msg.entropy.pt [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+ * BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+message Configure_entropy	{Configurar entropy}
```
Comment (hydrocat) [inline usr.sbin/sysinst/msg.entropy.pt:29]:
```suggestion

message Configure_entropy	{Configurar entropia}
```
Entropy não é nome de algum pacote né ?

File: usr.sbin/sysinst/msg.entropy.pt [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+message continue_without_entropy	{Agora nao, continue!}
+
+message not_enough_entropy
+{Este sistema parece nao ter um gerador de n�meros pseudoaleat�rios
+criptograficamente forte. Nao h� entropia suficiente dispon�vel para criar chaves
+seguras (por exemplo, chaves de host ssh).
+
+Se voc� planeja usar esta instala�ao para trabalho de produ�ao e
+por exemplo, ter� chaves de host ssh geradas, recomendamos fortemente concluir
+a configura�ao de entropia agora!
```
Comment (hydrocat) [inline usr.sbin/sysinst/msg.entropy.pt:40]:
```suggestion
Se voc� planeja usar esta instala�ao em produ�ao e
por exemplo, gerando chaves de ssh, recomendamos fortemente concluir
a configura�ao de entropia agora!
```

File: usr.sbin/sysinst/msg.entropy.pt [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+
+Voc� pode usar dados aleat�rios gerados em outro computador e carreg�-los
+aqui, ou pode inserir caracteres aleat�rios manualmente.
+
+Se voc� tiver um dispositivo de n�mero aleat�rio USB, conecte-o agora e selecione
+a op�ao "Reteste".}
+
+message entropy_add_manually		{Insira manualmente caracteres aleat�rios}
+message entropy_download_raw		{Carregar dados bin�rios aleat�rios}
+message	entropy_download_seed		{Importar arquivo de entropia do NetBSD}
```
Comment (hydrocat) [inline usr.sbin/sysinst/msg.entropy.pt:50]:
```suggestion
message	entropy_download_seed		{Importar um arquivo de entropia do NetBSD}
```

File: usr.sbin/sysinst/msg.entropy.pt [hunk 14]
```diff
@@ -0,0 +1,129 @@
...
+ */
+message entropy_via_nfs
+{Selecione um servidor, um compartilhamento e o caminho para carregar o $0.}
+
+/* Called with:				Example
+ *  $0 = content of file		NetBSD entropy seed file
+ */
+message entropy_via_download
+{Como nao h� entropia suficiente dispon�vel neste sistema, todas as opera�oes
+criptogr�ficas sao suspeitas de ataques de repeti�ao.
```
Comment (hydrocat) [inline usr.sbin/sysinst/msg.entropy.pt:107]:
```suggestion
criptogr�ficas sao sucetíveis de ataques de repeti�ao.
```
Sei que não é bem o que está escrito na verão em inglês (suspect), mas acho que faz mais sentido. O que você acha ?

File: usr.sbin/sysinst/msg.mbr.pt [hunk 15]
```diff
@@ -0,0 +1,140 @@
...
+{
+Voc� ser� solicitadoa informar a geometria.
+Insira o n�mero de stores por trilha (m�ximo 63)
+e o n�mero de cabe�as (m�ximo 256) queo BIOS usa para acessar o disco.
+O n�mero de cilindros ser� calculado a partir do tamanho do disco.
+
+}
+
+message nobiosgeom
+{Sysinst nao p�de determinar automaticamente a geometria do BIOS do disco.
```
Comment (hydrocat) [inline usr.sbin/sysinst/msg.mbr.pt:63]:
```suggestion
{Sysinst nao p�de determinar automaticamente a geometria da BIOS do disco.
```
`BIOS` é masculino ou feminio ? haha

File: usr.sbin/sysinst/msg.mbr.pt [hunk 16]
```diff
@@ -0,0 +1,140 @@
...
+{padrao}
+.endif
+
+message mbr_get_ptn_id {Tipo de parti�ao (0..255)}
+message Only_one_extended_ptn {S� pode haver uma parti�ao estendida}
+
+message mbr_flags	{ab}
+message mbr_flag_desc
+.if BOOTSEL
+{, (a)tivo, (b)oot padrao}
```
Comment (hydrocat) [inline usr.sbin/sysinst/msg.mbr.pt:91]:
```suggestion
{, (a)tivo, boot pa(d)rao}
```
Em casos como esse, a letra A e D são opções que os usuarios podem escolher ?
Se sim, devemos manter a grafia que indica a opção com a letra do inglês ?
Por exemplo, O usuário pode escolher a letra B, mas o código espera A ou D

File: usr.sbin/sysinst/msg.pm.pt (hunk 8 above)
Comment (hydrocat) [inline usr.sbin/sysinst/msg.pm.pt:58]:
```suggestion
message pmunused {(inutilizado)}
```
Sem uso também é bom. Fica a sugestão.

File: usr.sbin/sysinst/msg.pm.pt [hunk 17]
```diff
@@ -0,0 +1,167 @@
...
+message wannatry {voc� quer tentar?}
+message create_cgd {Criar volume criptogr�fico (CGD)}
+message create_vnd {Criar imagem de disco virtual (VND)}
+message create_vg {Criar grupo de volume (LVM VG)}
+message create_lv {Criar volume l�gico}
+message create_raid {Criar RAID por software}
+message updpmlist {Atualizar lista de dispositivos}
+message savepm {Salvar mudan�as}
+message pmblocked {BLOQUEADO}
+message pmunchanged {SEM USO}
```
Comment (hydrocat) [inline usr.sbin/sysinst/msg.pm.pt:55]:
```suggestion
message pmunchanged {INALTERADO}
```

em inglês, temos: `message pmunchanged {UNCHANGED}`. para mim soa como "Não alterado" ao invés de "Sem uso".
No tradutor, a sugestão foi de "Inalterado", acho bom, mas talvez exista um termo melhor.

**Review by hydrocat — COMMENTED** [review]
Adicionei algumas sugestões de maneira que fica fácil você aplicar pela interface do GitHub.

File: usr.sbin/sysinst/msg.mi.pt [hunk 18]
```diff
@@ -0,0 +1,1528 @@
...
+Ative a sele�ao atual do menu digitando a tecla Enter.
+}
+
+message thanks
+{Obrigado por usar o NetBSD!
+}
+
+message installusure
+{Voc� escolheu instalar o NetBSD no seu disco r�gido. Isso mudar�
+as informa�oes no seu disco r�gido. Voc� deveria ter feito um backup completo
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.mi.pt:85]: "deveria" isn't kind of a weird verbal time here? How about rephrasing it as "É suposto que já exista um backup completo"?

File: usr.sbin/sysinst/msg.mi.pt [hunk 19]
```diff
@@ -0,0 +1,1528 @@
...
+antes deste procedimento! Este procedimento far� o seguinte:
+	a) Particionar seu disco
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.mi.pt:87]: Given that you're only saying "fará o seguinte", perhaps "Particionará" would make more sense?

File: usr.sbin/sysinst/msg.mi.pt [hunk 20]
```diff
@@ -0,0 +1,1528 @@
...
+ *  $2 = short version of $1		GPT
+ *  $3 = disk size for NetBSD		3TB
+ *  $4 = full install size min.		127M
+ *  $5 = install with X min.		427M
+ */
+message layout_main
+{
+Voc� ter� entao a oportunidade de alterar qualquer um dos detalhes da parti�ao
+
+A parte NetBSD (ou livre) do seu disco ($0) custa $3.
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.mi.pt:200]: Instead of "custa", wouldn't "consome" be more appropriate?

File: usr.sbin/sysinst/msg.mi.pt [hunk 21]
```diff
@@ -0,0 +1,1528 @@
...
+escolher tamanhos de parti�ao que sejam m�ltiplos exatos do seu tamanho real
+de cilindro.
+
+Escolha seu especificador de tamanho}
+
+message ptnsizes
+{Agora voc� pode alterar os tamanhos das parti�oes do sistema. O padrao �
+alocar todo o espa�o para o sistema de arquivos raiz. No entanto, voc� pode desejar
+ter sistemas de arquivos /usr (arquivos de sistema adicionais), /var (arquivos de log etc.)
+ou /home (diret�rios home dos usu�rios) separados.
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.mi.pt:220]: I suspect this "home"  in the parenthesis is not really helping much with the semantics?

File: usr.sbin/sysinst/msg.mi.pt [hunk 22]
```diff
@@ -0,0 +1,1528 @@
...
+{Ok, agora estamos prontos para instalar o NetBSD no seu disco r�gido (%s). Nada foi
+escrito ainda. Esta � sua �ltima chance de sair deste processo antes que algo
+seja alterado.
+
+Vamos continuar?
+}
+
+message disksetupdone
+{Ok, a primeira parte do procedimento est� conclu�da. O Sysinst
+escreveu um disklabel no disco de destino e fez newfs'ed e fsck'ed
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.mi.pt:416]: Perhaps "completou a criação e checagem das partições com newfs e fsck" would make more sense for people not that familiar with English?

File: usr.sbin/sysinst/msg.mi.pt [hunk 23]
```diff
@@ -0,0 +1,1528 @@
...
+Alguma delas se conecta ao servidor necess�rio?}
+
+message asknetdev
+{Qual dispositivo de rede voc� gostaria de usar?}
+
+message netdevs
+{Interfaces dispon�veis}
+
+message netinfo
+{To be able to use the network, we need answers to the following:
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.mi.pt:594]: You might have forgotten to translate this.
My suggestion: "A fim de poder utiliar a rede, são necessárias respostas para as seguintes questões:"

File: usr.sbin/sysinst/msg.mi.pt [hunk 24]
```diff
@@ -0,0 +1,1528 @@
...
+}
+
+message abortunpack
+{Descompactar conjuntos adicionais nao foi bem-sucedido. Voc� precisar�
+continuar manualmente ou escolher uma fonte diferente para conjuntos de lan�amento e tentar
+novamente.
+}
+
+message createfstab
+{TH� um grande problema! Nao � poss�vel criar /mnt/etc/fstab. Saindo!
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.mi.pt:722]: I think your finger slipped here and you typed an extra T? Or maybe you forgot to delete a previously existing one?

File: usr.sbin/sysinst/msg.mi.pt [hunk 25]
```diff
@@ -0,0 +1,1528 @@
...
+%s/bin/X.old for de uma atualiza�ao conclu�da, voc� pode rm -f
+%s/bin/X.old e reiniciar. Ou se %s/bin/X.old for de
+uma atualiza�ao recente e incompleta, voc� pode rm -f %s/bin/X e mv
+%s/bin/X.old para %s/bin/X
+
+Abortando a atualiza�ao.}
+
+message netnotup
+{Houve um problema na configura�ao da rede. Ou seu gateway
+ou seu servidor de nomes nao estava acess�vel por um ping. Voc� quer
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.mi.pt:750]: I'm far from being an expert in Portuguese, but shouldn't "estava acessível" be in plural here, to match the gateway and name server?

File: usr.sbin/sysinst/msg.mi.pt [hunk 26]
```diff
@@ -0,0 +1,1528 @@
...
+* TODO
+*/
+/* Called with:				Example
+ *  $0 = device name			/dev/rwd0a
+ *  $1 = file system type		ffs
+ *  $2 = error return code form fsck	8
+ */
+message badfs
+{It appears that $0 is not a $1 file system or the fsck was
+not successful.  Try mounting it anyway?  (Error number $2.)
```
Comment (TheFakeMontyOnTheRun) [inline usr.sbin/sysinst/msg.mi.pt:774]: I suppose this is where your WIP currently stops. Thank you for all the effort so far!
Hope I don't sound pedantic or picky with my suggestions. I just want to help you make the best damn BSD translation we can :)

File: usr.sbin/sysinst/msg.mi.pt (hunk 18 above)
Comment (hydrocat) [reply usr.sbin/sysinst/msg.mi.pt:85]: I believes that this comes from the English `You should have made a full backup before this procedure!` which is also in the past.

File: usr.sbin/sysinst/msg.mi.pt (hunk 21 above)
Comment (hydrocat) [reply usr.sbin/sysinst/msg.mi.pt:220]: How would you translate home directory ?
Personally this is part of the computer guy's jargon, but a good translation is always welcome.

File: usr.sbin/sysinst/msg.mi.pt (hunk 22 above)
Comment (hydrocat) [reply usr.sbin/sysinst/msg.mi.pt:416]: Isso ou `[...] destino e executou newfs e fsck`

File: usr.sbin/sysinst/msg.entropy.pt (hunk 11 above)
Comment (TheFakeMontyOnTheRun) [reply usr.sbin/sysinst/msg.entropy.pt:29]: Não que eu saiba - meu entendimento é o do conceito de entropia mesmo.

File: usr.sbin/sysinst/msg.mi.pt (hunk 21 above)
Comment (TheFakeMontyOnTheRun) [reply usr.sbin/sysinst/msg.mi.pt:220]: I'm completely OK with home. My point was more in the sense of the extra remark being a bit redundant.

File: usr.sbin/sysinst/msg.mbr.pt (hunk 15 above)
Comment (TheFakeMontyOnTheRun) [reply usr.sbin/sysinst/msg.mbr.pt:63]: É "o software", não? Mas suspeito que qualquer um dos dois serviria perfeitamente.

**Pull request closed without merging**